index.html

```diff
@@ -246,17 +346,19 @@ <h4>Skills</h4>
                   <li>Terminal</li>
                 </ul>
               </li>
+              <li class="skill-item-bg-decoration hide"></li>
             </ul>
           </div>
         </div>
       </section>
 
       <!-- Footer Section -->
       <footer class="section footer-section bg-decoration">
-        <div class="container">
+        <div class="container grid">
+          <div class="top-bg-decoration hide"></div>
           <div class="top-footer-container flex direction">
             <div class="contact-content">
-              <h2 class="primary-heading">Get started</h2>
+              <h2>Get started</h2>
```

Comment:
- [ ] Please correct the style for the "Get started". It's creating a margin when responsive. Your portfolio must look amazing for any size of screen.
<img width="779" alt="Screen Shot 2022-08-27 at 11 59 54 AM" src="https://user-images.githubusercontent.com/47673311/187035780-6fbeb86c-24e3-4d31-995e-549fe8ea703b.png">